src/Modules/Processing/CustomPlayer/CustomPlayer.cpp

```diff
@@ -37,9 +38,42 @@ void CustomPlayer::exec() {
   if (!field || !frame || !robot) {
     return;
   }
+  if(!frame->has_ball()) return;
+  auto&& pos_ball = frame->ball();
+  if(field->enemyPenaltyAreaContains(pos_ball)) return;
+  if(robot->distTo(pos_ball)>150)
+  {
+    SSLMotion::GoToPoint goToPoint(pos_ball,
+                                   (pos_ball - robot->position()).angle(),
+                                   true);
+    auto command = sslNavigation.run(robot.value(), SSLRobotCommand(goToPoint));
+    command.set_dribbler(true);
+    command.set_dribblerVelocity(4);
+    emit sendCommand(command);
+  }else {
+    SSLMotion::RotateOnSelf Rotate((field->enemyGoalInsideCenter() -robot->position()).angle());
+    auto command = sslNavigation.run(robot.value(), SSLRobotCommand(Rotate));
+    command.set_dribbler(true);
+    command.set_dribblerVelocity(4);
+    emit sendCommand(command);
+    if(abs(robot->angleTo(field->enemyGoalInsideCenter()))<0.1)
+    {
+      SSLMotion::GoToPoint goToPoint(pos_ball,
+        (pos_ball - robot->position()).angle(),
+        true);
+      auto irparapos = sslNavigation.run(robot.value(), SSLRobotCommand(goToPoint));
+      irparapos.set_front(true);
+      irparapos.set_kickSpeed(2);
```

Comment:
Pode ser um nome mais descritivo
```suggestion
      auto command = sslNavigation.run(robot.value(), SSLRobotCommand(goToPoint));
      command.set_front(true);
      command.set_kickSpeed(2);
```